devel/build

```diff
@@ -19,7 +19,14 @@ export GIT_REVISION=$(git describe --tags --abbrev=40 --always --dirty || true)
 # Calling `docker run nextstrain/base` will still only pull down the small base
 # image rather than pulling down the larger nextstrain/base-builder image.
 
-docker build \
+platform=linux/amd64
+
+# `buildx create` is necessary to use a driver that supports multi-platform
+# images.
```

Comment:
At first glance I didn't understand why we need to always `create`, but this comment helped it make sense, so that's nice. :-) (Assuming I understand correctly that the default `docker` builder is insufficient compared to the `docker-container` builder that `create` gives us by default.)

Reply:
That's also my understanding, though I couldn't find explicit documentation that states:

1. `docker buildx build` uses `docker` driver by default
2. `docker buildx create` uses non-`docker` driver by default

[The `--driver` section of `docker buildx create` docs](https://docs.docker.com/engine/reference/commandline/buildx_create/#driver) is probably the most relevant.

Reply:
Done:

https://github.com/nextstrain/docker-base/blob/40339783f16d3c76901bb13011467f40aaed6f34/devel/build#L26